### ISSUE TITLE
UX feedback

### ISSUE THREAD

**Issue by rofe** [adobe/helix-bulk-editor#78]
Bulk Editor Enhancement

**Home/main screen**

- [ ] use 2 centered, large "cards" with icons instead of text links for the 2 main actions, export and import

| Export | Import |
| :---: | :---: |
| [icon] | [icon] |
| Export data to a CSV file | Import data from CSV file  |

**Export CSV**

- [ ] _Select Folder_ dialog: browse to a folder and click _OK_, then click _Browse_ again to change the folder. Now navigating back up from a previously selected folder isn't possible anymore. There's only a small _/_ character which, once you're able to click it, does nothing. Workaround: reload page and start over
- [ ] _Extract_ button: this action should happen automatically after selecting an export path, so this button can be removed
- [ ] Once extraction is complete, a table with the extracted documents is shown. This tabe shouls use the full remaining viewport height to minimize the need for scrolling.
- [ ] Why is `en/publish` cut off in the path property of each doucment? This could be confusing because when another user looks at the CSV later on, they don't necessarily know the context of where these documents came from. Also, when selecting a different folder (e.g. `en/topics`), no path gets displayed at all. Proposed solution: keep the full path from the root folder.
- [ ] _Get CSV_ button could be renamed to just Download_.
- [ ] Once the user has downloaded the CSV file, the table remains visible. There is nothing the user can do here, so the UI might as well show a modal dialog confirming that the download of n documents is complete, and upon clicking _OK_ take the user back to the main screen.
- [ ] Add Word support

**Import CSV**

- [ ] _Reset_ button: cancels the import and goes back to the main screen, so should probably be titled "Cancel" (and not highlghted in red, because no data is being destroyed)
- [ ] _Verify_ button: similar to the _Extract_ button during the export, this action should happen automatically upon loading a CSV file, so this button can be removed
- [ ] 50x errors: if an error occurs during the verification step, a box is shown at the top of the table. It is currently not clear to the user what to do about it. Proposal: The error could be disppayed in a modal dialog and have 2 buttons: _Cancel_ (go back to the main screen) and _Retry_ (trigger the verification step again).
- [ ] Once verification is complete, the table shows the diffs in cells that will be changed, which is neat. If there are many rows and only few changes however, it can be quite tedious to scroll through the entire table to find all the diffs. It would therefore be nice to have a _Hide unchanged_ or _Show only changes_ checkbox at the top (or bottom), and maybe even select it by default.
- [ ] _Update Documents_ button: shows a modal dialog asking the user to confirm the change. Here is would be nice to say how many documents will be updated, to give the user an idea about the scope of the changes.
- [ ] After updating documents is complete, the table remains visible with just the unchanged documents. There is nothing the user can do here, so the UI might as well show a modal dialog confirming that the update of n documents is complete, and upon clicking _OK_ take the user back to the main screen.
